README.md

```diff
@@ -55,6 +55,12 @@ The flask app has now been altered so that it is now partially integrated to the
 
 -A Trello Account to access the Trello API (This is free)
 
--To set up the Trello enviromental variables (Key, Token, List IDs etc) either by the altering the DevOps-Course-Starter/.env file or creating a seperate config file with those variables inside
+-A .env file with the Trello Enviromental variables (Key, Token, List IDs etc) inside, as well as a Flask Secret Key as well as the server configuration
 
---If a new config file is created then one needs to insert that file in the .gitignore file so that it is not committed
\ No newline at end of file
+In order to test if the app is working, type into the command line 
```

Comment:
Looks like we're missing a couple of prerequisites here - you've correctly git-ignored the geckodriver.exe file, but that means a new developer won't be able to run the end-to-end tests without that so we should raise that explicitly, and also worth noting that there's a dependency on firefox that isn't specified